partcad/src/partcad/project.py

```
@@ -25,6 +26,7 @@
 from . import sketch
 from . import sketch_config
 from .exception import EmptyShapesError
+from .exception import EmptyShapesError
```

Ruff (0.8.2)

Redefinition of unused `EmptyShapesError` from line 28

Remove definition: `EmptyShapesError`

(F811)